### ISSUE TITLE
Create category modal

### ISSUE THREAD

**Issue by carlosvirreira** [Shelf-nu/shelf.nu#121]
Right now the UX is good. But I think that what Clockify does with a modal is way better.

<img width="998" alt="Screenshot 2023-05-16 at 12 40 17" src="https://github.com/Shelf-nu/shelf.nu/assets/12449255/1abae836-e870-4242-b547-799f9dc30c06">

**Comment by jurrejansen** [comment]
Yes, There is a 'Create Category' flow on the 'Index' page in Figma. How it currently works is not ideal when user wants create a new category whilst in the process of creating an asset.

![Screenshot 2023-06-01 at 11 07 56](https://github.com/Shelf-nu/shelf.nu/assets/76112272/a983b17b-849e-4656-a806-c37b853ab6b7)
